portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -3658,6 +3658,55 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-126479."
+	@priority = "4"
+	test SubmitFormWhenSessionExpires {
+		property portal.acceptance = "true";
```

Comment:
@mlan0, please set this property to false.

Reply:
Done.